main/model/config.py

```diff
@@ -26,7 +26,7 @@ class Config(model.Base, model.ConfigAuth):
   recaptcha_private_key = ndb.StringProperty(default='', verbose_name='Private Key')
   recaptcha_public_key = ndb.StringProperty(default='', verbose_name='Public Key')
   salt = ndb.StringProperty(default=util.uuid())
-  trusted_hosts = ndb.StringProperty(repeated=True, verbose_name='Trusted hosts')
+  trusted_hosts = ndb.StringProperty(repeated=True, verbose_name='Trusted Hosts')
```

Comment:
Even though lately I prefer the sentence case we'll have to keep it consistent :)

Here is an interesting post about it.. 
https://medium.com/@jsaito/making-a-case-for-letter-case-19d09f653c98